app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarController.kt

```diff
@@ -254,6 +284,13 @@ class DefaultBrowserToolbarController(
             ToolbarMenu.Item.SaveToCollection -> Event.BrowserMenuItemTapped.Item.SAVE_TO_COLLECTION
             ToolbarMenu.Item.AddToHomeScreen -> Event.BrowserMenuItemTapped.Item.ADD_TO_HOMESCREEN
             ToolbarMenu.Item.Quit -> Event.BrowserMenuItemTapped.Item.QUIT
+            is ToolbarMenu.Item.ReaderMode ->
+                if (item.isChecked) Event.BrowserMenuItemTapped.Item.READER_MODE_ON
```

Comment:
vvvv small nit: can we put { } around these "then" statements? I believe we do this everywhere else we have them. I know they're not necessary, but it could lead to someone accidentally changing it down the road